solidity/test/BancorNetwork.js

```diff
@@ -157,7 +154,7 @@ describe('BancorNetwork', () => {
         const converterFactory = await ConverterFactory.new();
         await contractRegistry.registerAddress(registry.CONVERTER_FACTORY, converterFactory.address);
 
-        const networkSettings = await NetworkSettings.new(NETWORK_FEE_WALLET, NETWORK_FEE);
+        const networkSettings = await NetworkSettings.new(defaultSender, 0);
```

Comment:
It's not what I had in mind, but that's fine for now. We will change it soon anyway.

Reply:
There is no purpose to test the network-fee here, just make the system operable.

Reply:
Gotcha